pkg/controller/migcluster/migcluster_controller.go

```diff
@@ -117,6 +122,8 @@ func (r *ReconcileMigCluster) Reconcile(request reconcile.Request) (reconcile.Re
 
 	// Report reconcile error.
 	defer func() {
+		log.Info("CR", "conditions", cluster.Status.Conditions)
```

Comment:
Didn't notice this on the last PR but:
Thinking something like this would be clearer:
```
log.Info("Conditions", "list", cluster.Status.Conditions)
```
The "CR" feels cryptic.

Reply:
I am not sure about this one. I agree it feels cryptic in the code, but if you look at the logs entry it produces, it is in the format

```
{... "CR", "cluster", "foo-ns/foo-cluster", "conditions", "[]"}
```

The suggested change will look like,

```
{... "Conditions", "cluster", "foo-ns/foo-cluster", "list", "[]"}
```
The former seems more readable log entry IMO